### ISSUE TITLE
Build and upload automatically docker images

### ISSUE THREAD

**Issue by MuriloChianfa** [secureworks/dalton#162]
I followed the installation tutorial in the readme, and when i build the docker images they take a long time to compile all images.

The ideia of this issue is to suggest that we use Github actions to build and auto upload to Docker Hub all the images, and cut down the time of image compilations.

It would only be necessary to download them, changing the command:
```bash
docker compose build && docker compose up -d
```
to:
```bash
docker compose pull && docker compose up -d
```

Or even add the pull_policy: pull statement so you don't need 2 commands, with this command, its already download the images automatically if they not founded locally:
```bash
docker compose up -d
```

**Comment by whartond** [comment]
Certainly, using prebuilt images would be faster.  However, the current strategy is intentional -- by building starting with "official" docker images, the integrity, trustworthiness, and transparency of the system is emphasized, rather than encouraging the download and running of arbitrary containers which the user has very little insight and assurance as to what they actually contain and do.

Docker's build cache means that after the initial build, startup should be much quicker.

If building is taking a long time and you aren't using the Zeek feature(s), I suggest commenting out the Zeek build(s) in docker-compose.yml since Zeek typically takes an outsized time to build.
